login-workflow/src/pages/create-account/steps/verify-email/verify-email.component.ts

```diff
@@ -40,13 +40,18 @@ import { BluiAuthTranslations } from '../../../../translations/auth-translations
                     </mat-error>
                 </mat-form-field>
             </form>
-            <button
-                mat-flat-button
-                color="primary"
-                style="width: 100%; margin-top: 8px;"
-                (click)="sendVerificationEmail()"
-                [innerHTML]="translate.CREATE_ACCOUNT.VERIFY_EMAIL.RESEND_BUTTON"
-            ></button>
+            <div style="display: flex; flex-direction: row;">
+                <div class="mat-body-1">{{ translate.CREATE_ACCOUNT.VERIFY_EMAIL.RESEND_LABEL }}</div>
+                <div
+                    color="primary"
+                    class="blui-auth-link mat-subheading-1"
+                    id="blui-resend-button-link"
+                    style="line-height: inherit;"
+                    (click)="sendVerificationEmail()"
+                >
+                    &nbsp;<u>{{ translate.CREATE_ACCOUNT.VERIFY_EMAIL.RESEND_BUTTON }}</u>
+                </div>
```

Comment:
Why is this not a button or anchor element? The `&nbsp;` here adds 1 character whitespace at the front of the translation which isn't terrible, but it may have been more practical to have the spacing handled with margin so that there isn't an empty, clickable space.